build/generate-manifests.sh

```diff
@@ -143,6 +154,30 @@ $RELATED_IMAGES"
     sed -i -e 's+__REPLACES_CSV__+'"  replaces: $OPERATOR_NAME.v$REPLACES_CSV"'+g' "${csv_path}"
   fi
 
+  if [[ ${COMMUNITY} == "true" ]]; then
+    sed -i '/__OLM_FEATURE_DISCONNECTED__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_FIPS_COMPLIANT__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_PROXY_AWARE__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_TLS_PROFILES__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_TOKEN_AUTH_AWS__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_TOKEN_AUTH_AZURE__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_TOKEN_AUTH_GCP__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_CNF__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_CNI__/d' "${csv_path}"
+    sed -i '/__OLM_FEATURE_CSI__/d' "${csv_path}"
```

Comment:
Do we really need to remove these from the community CSV? There's nothing wrong with including these annotations in the community CSV.

Reply:
I see. Yes, I believe the community version should have different values (e.g. it isn't FIPS compliant and doesn't support disconnected installs). 

But I've also just realized that the whole point of this PR is to add all these annotations, since they are required. If they are required, they are required in both the product and the community CSV, right?